multiagenticswarm/core/prompt_parser.py

```diff
@@ -0,0 +1,204 @@
+# multiagenticswarm/core/prompt_parser.py
+import re
+import json
+import os
+from typing import Dict, Any, Optional, Callable
+from langchain_core.messages import SystemMessage, HumanMessage
+from langchain_groq import ChatGroq
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+
+# Import schema + role mappings
+from prompt_schemas import DEFAULT_FUNCTION_SCHEMA, ROLE_MAPPINGS
+print(DEFAULT_FUNCTION_SCHEMA)
```

Comment:
Never use print() in library code. Use proper logging

We have dedicated logging for this library, please use that